packages/lesswrong/components/posts/PostsPage/PostsPageTitle.jsx

```diff
@@ -5,26 +5,32 @@ import Typography from '@material-ui/core/Typography';
 import { Link } from 'react-router-dom';
 import { Posts } from '../../../lib/collections/posts';
 
-const styles = theme => ({
-  root: {
-    ...theme.typography.display3,
-    ...theme.typography.postStyle,
-    ...theme.typography.headerStyle,
-    margin: "0 !important",
-    color: theme.palette.text.primary,
-    [theme.breakpoints.down('sm')]: {
-      fontSize: '2.5rem',
+const styles = theme => {
```

Comment:
This seems unlikely to warrant such special casing.

Reply:
Yep and we don't even have bold titles any more. Reverted.